app/models/form/sales/pages/about_price_value_check.rb

```diff
@@ -7,7 +7,7 @@ def initialize(id, hsh, subsection)
       },
     ]
     @title_text = {
-      "translation" => "soft_validations.purchase_price.title_text",
+      "translation" => "forms.#{form.start_date.year}.#{@copy_key}.title_text",
```

Comment:
Does this (and other soft validation pages) not need a copy key specifying? (Because the translations are under soft_validations rather than the subsection)